src/components/ReactTags.js

```diff
@@ -398,6 +401,9 @@ class ReactTags extends Component {
       inputId = this.props.id,
       maxLength = this.props.maxLength;
 
+    // read inputFieldPosition
+    const inputFieldPosition = !this.props.inline ? INPUT_FIELD_POSITIONS.BOTTOM : this.props.inputFieldPosition;
```

Comment:
can you destruct `inline` and `inputFieldPosition` above and use it here and change this line to
```
const position = !inline ? INPUT_FIELD_POSITIONS.BOTTOM : inputFieldPosition;
```